incydr/cli/cmds/alerts.py

```diff
@@ -64,7 +66,7 @@ def search(
     output: Optional[str],
     certs: Optional[str],
     ignore_cert_validation: Optional[bool],
-    advanced_query: Optional[str],
+    advanced_query: Optional[Union[str, File]],
```

Comment:
should we just use the `FileOrString` type here?

Reply:
Oops, looks like we already are.. I forgot I had modified the `FileOrString` to not read the file automatically when I brought it over from the code42cli...

Reply:
Yeah I was just adjusting the type hint, I'm sure I could set up a callback in the `FileOrString` option but its pretty simple to just do it within the method.